tests/e2e/fixtures/imagevolume_extensions/cluster-with-extensions.yaml.template

```diff
@@ -0,0 +1,36 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: postgresql-with-extensions
+spec:
+  instances: 3
+
+  postgresql:
+    parameters:
+      max_connections: "110"
+      log_checkpoints: "on"
+      log_lock_waits: "on"
+      log_min_duration_statement: '1000'
+      log_statement: 'ddl'
+      log_temp_files: '1024'
+      log_autovacuum_min_duration: '1s'
+      log_replication_commands: 'on'
+    extensions:
+      - name: postgis
+        ld_library_path:
+          - system
+        image:
+          reference: ghcr.io/niccolofei/postgis:18beta1-master-bullseye # wokeignore:rule=master
```

Comment:
We'll need to remove this hardcoded reference from here.  These tests are run with different PG major versions.